linalg/vector.cpp

```diff
@@ -68,6 +68,12 @@ void Vector::Load(std::istream **in, int np, int *dim)
       for (j = 0; j < dim[i]; j++)
       {
          *in[i] >> data[p++];
+#if defined __APPLE__ && defined __clang__
```

Comment:
Perhaps we should change this to something like `#ifdef _LIBCPP_VERSION` to detect libc++, which appears to be the source of the issue.

Reply:
Why would we need this macro in the first place? If we do not want this code in release builds for performance reasons, then a debug build check should be enough. However I would not expect this additional branch to affect the performance anyway.

Reply:
I would be fine disabling this `#if` check altogether. I'd prefer not to make it debug only.

Reply:
I don't think one check (in the normal case) is going to have any measurable performance effect. I think we can always do the check (i.e. remove the `#if ... #endif`) just to be safe. I'm not sure what the effect will be on `Inf`s or `Nan`s which are not read correctly in any case, I think.

Another point: zeroing out denormals during output may not be sufficient by itself -- due to round-off during conversion to text some small normal numbers may become denormals when read back. So I think we can remove the zeroing of denormals when writing to a stream.

Reply:
I removed the `#if`, I agree that this solution makes the most sense.

I think zeroing out the subnormals during output is still useful (even if there are some corner cases where it may not be sufficient). I added this also to the ParaView output, because ParaView has the same issue when reading ASCII files.

Reply:
I guess it make sense to zero out denormals for paraview at least.